src/components/ContentHighlights/HighlightStepper/HighlightStepperConfirmContent.jsx

```diff
@@ -7,54 +9,57 @@ import {
   Col,
   Icon,
   CardGrid,
+  Alert,
 } from '@edx/paragon';
 import { Assignment } from '@edx/paragon/icons';
 import { camelCaseObject } from '@edx/frontend-platform';
 import { Configure, InstantSearch, connectStateResults } from 'react-instantsearch-dom';
-
 import { configuration } from '../../../config';
-import { STEPPER_STEP_TEXT, MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET } from '../data/constants';
-import { ContentHighlightsContext } from '../ContentHighlightsContext';
-import SkeletonContentCard from '../SkeletonContentCard';
 
-const prodEnterpriseId = 'e783bb19-277f-479e-9c41-8b0ed31b4060';
+import {
+  STEPPER_STEP_TEXT,
+  MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET,
+  HIGHLIGHTS_CARD_GRID_COLUMN_SIZES,
+  DEFAULT_ERROR_MESSAGE,
+} from '../data/constants';
+import { ContentHighlightsContext } from '../ContentHighlightsContext';
+import ContentConfirmContentCard from './ContentConfirmContentCard';
+import SkeletonContentCardContainer from '../SkeletonContentCardContainer';
 
-const BaseReviewContentSelections = ({
+export const BaseReviewContentSelections = ({
   searchResults,
   isSearchStalled,
 }) => {
   if (isSearchStalled) {
     return (
-      <CardGrid
-        columnSizes={{
-          xs: 12,
-          md: 6,
-          lg: 4,
-          xl: 3,
-        }}
-      >
-        {[...new Array(8)].map(() => <SkeletonContentCard />)}
-      </CardGrid>
+      <SkeletonContentCardContainer length={MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET} />
     );
   }
-
   if (!searchResults) {
-    return null;
+    return (<div data-testid="base-content-no-results" />);
   }
-
   const { hits } = camelCaseObject(searchResults);
 
+  const transformedCardData = hits.map((highlightedContent) => {
```

Comment:
[curious] Is this variable necessary versus doing `hits.map(highlightedContent => <ContentConfirmContentCard key={highlightedContent.aggregationKey} original={highlightedContent} />)` on line 60 below?